### PR TITLE
Fix spelling mistake

### LINKED ISSUE #1471: Fix the typo in the success stories page
**Is your feature request related to a problem? Please describe.**
- There's a typo in the success stories and scholars pages

**Describe the solution you'd like**
- Alumini should be alumni

Screenshot 2022-12-11 at 9 06 08 AM

**Additional context**
Try to find and replace it because there might be other places with the same mistake

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by dilankavishka**
## Purpose

The purpose of this PR is to fix #1471 

## Goal
To fix the spelling mistake in ScholarX page and ScholarX success Stories section.

## Approach

Fixed the spelling mistake in ScholarX page and ScholarX success Stories section.

### Screenshots
Screenshot 2022-12-15 at 19 58 19

Screenshot 2022-12-15 at 22 06 37

  
### Preview Link

https://pr-1476-sef-site.surge.sh/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by YoshithaRathnayake — CHANGES_REQUESTED** [review]
There is a word `alumini` in the success-stories page @dilankavishka 

https://github.com/sef-global/sef-site/blob/bad025879edd857f31680f1c9b823974a5cb6b2f/scholarx/success-stories/index.html#L162

image

Change it to `alumni`

File: scholarx/2022/index.html [hunk 1]
```diff
@@ -170,7 +170,7 @@ <h1 class="display-3 text-center mb-5">Program Timeline</h1>
             <div class="row justify-content-center">
                 <div class="col-11 text-center">
                     <p class="display-3 text-onelive font-weight-bolder mb-0">Let's hear it from our</p>
-                    <p class="display-3 text-onelive font-weight-bolder">ScholarX Alumini</p>
+                    <p class="display-3 text-onelive font-weight-bolder">ScholarX Alumni</p>
```
Comment (anjula-sack) [inline scholarx/2022/index.html:173]: Are you sure there are no other places with this mistake? @dilankavishka

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html:173]: @anjula-sack there is a one in the success-stories page

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html:173]: https://github.com/sef-global/sef-site/pull/1476#pullrequestreview-1219456498

Comment (dilankavishka) [reply scholarx/2022/index.html:173]: I'm corrected the other misspelling word also. @anjula-sack @YoshithaRathnayake

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html:173]: https://github.com/sef-global/sef-site/pull/1476#pullrequestreview-1219456498 here you are not corrected it @dilankavishka

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html:173]: There is a word `alumini` in the success-stories page @dilankavishka 

https://github.com/sef-global/sef-site/blob/bad025879edd857f31680f1c9b823974a5cb6b2f/scholarx/success-stories/index.html#L162

image

Change it to `alumni`

Comment (dilankavishka) [reply scholarx/2022/index.html:173]: I changed it. @YoshithaRathnayake

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html:173]: But the changes are not committed @dilankavishka

Comment (YoshithaRathnayake) [reply scholarx/2022/index.html:173]: Check the preview link it's not corrected https://pr-1476-sef-site.surge.sh/scholarx/success-stories

**Review by YoshithaRathnayake — APPROVED** [review]
Thank You! @dilankavishka

**Comment by dilankavishka** [comment]
You're Welcome @YoshithaRathnayake

**Pull request merged**